src/sagemaker/image_uri_config/xgboost.json

```diff
@@ -1,7 +1,9 @@
 {
     "scope": ["inference", "training"],
     "version_aliases": {
-        "latest": "1"
+        "latest": "1", 
+        "0.90": "0.90-2", 
```

Comment:
In SDK v1, `0.90` pointed to `0.90-1` not `0.90-2`, because 1) 0.90 was released before we finalized our versioning story, and 2) the alias gets created when `-1` version is released, so for backwards compatibility, we are forced to alias the bare xgboost version to the very first container version. For backwards compatibility, the alias `0.90` needs to be pointed to `0.90-1` forever even if there is a `0.90-2`. With the major version bump to SDK v2, this may be okay for `0.90` or `1.0`, but this is why I don't think it's best to use the xgboost versions as aliases.